README.rst

```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
 ===========================================================
 
 .. image:: https://mybinder.org/badge_logo.svg
-  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
\ No newline at end of file
+  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
+
+Build Turgon on OS X
+====================
+Clone from https://github.com/yungyuc/turgon.git:
+ `$ git clone https://github.com/yungyuc/turgon.git`
```

Comment:
For code, please use `code-block`: https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/directives.html#directive-code-block.  There are several places to update.